<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generador de Diplomas</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: white;
background-color: rgb(132, 131, 130);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>81</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255)
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lbLogo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../icons/csoftmty_logo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lbEventName">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Nombre del evento</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignHCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="tfName">
           <property name="minimumSize">
            <size>
             <width>660</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Yu Gothic UI</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">color: black;
background-color: white;
border-radius: 5px;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lbFieldsAndDesign">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Diseño de Campos</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignHCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbName">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Nombre</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hlName">
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnColorName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Yu Gothic UI</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(237, 108, 50);
border-radius: 10px</string>
           </property>
           <property name="text">
            <string>Color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbFontName">
           <property name="font">
            <font>
             <family>Yu Gothic UI</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox{
    border: none;
    border-radius: 5px;
    background-color: rgb(255, 255, 255);
    color: rgb(0, 0, 0);
    padding: 5px; 
}

QComboBox::drop-down{
    background-color: rgb(237, 108, 50);
    border: none;
    width: 25px;
    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
}

QComboBox::down-arrow{
    image: url(package/icons/chevron.png);
    width: 14px;
    height: 14px;
}

QListView{
    border: none;
    color: rgb(132, 131, 130);
    background-color: rgb(255, 255, 255);
    font-weight: bold;
    selection-background-color: rgb(237, 108, 50);
    show-decoration-selected: 1;
    margin-left: -10px;
    padding-left: 15px;
}

QListView::item:hover{
    background-color: rgb(237, 108, 50);
    border: none;
}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbSizeName">
           <property name="font">
            <font>
             <family>Yu Gothic UI</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QSpinBox {
    border: none;
    border-radius: 5px;
    background-color: rgb(255, 255, 255);
    color: rgb(0, 0, 0);
    padding: 5px; 
}

QSpinBox::up-button {
    background-color: rgb(237, 108, 50);
    border: none;
    width: 25px;
    border-top-right-radius: 5px;
}

QSpinBox::down-button {
    background-color: rgb(237, 108, 50);
    border: none;
    width: 25px;
    border-bottom-right-radius: 5px;
}

QSpinBox::up-arrow {
    image: url(package/icons/chevron-up.png);
    width: 14px;
    height: 14px;
}

QSpinBox::down-arrow {
    image: url(package/icons/chevron.png);
    width: 14px;
    height: 14px;
}</string>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="value">
            <number>16</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lbDescription">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Descripción</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="tfDescription">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;
background-color: white;
border-radius: 5px;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hlDescription">
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnColorDesc">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Yu Gothic UI</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(237, 108, 50);
border-radius: 10px</string>
           </property>
           <property name="text">
            <string>Color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbFontDesc">
           <property name="font">
            <font>
             <family>Yu Gothic UI</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox{
    border: none;
    border-radius: 5px;
    background-color: rgb(255, 255, 255);
    color: rgb(0, 0, 0);
    padding: 5px; 
}

QComboBox::drop-down{
    background-color: rgb(237, 108, 50);
    border: none;
    width: 25px;
    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
}

QComboBox::down-arrow{
    image: url(package/icons/chevron.png);
    width: 14px;
    height: 14px;
}

QListView{
    border: none;
    color: rgb(132, 131, 130);
    background-color: rgb(255, 255, 255);
    font-weight: bold;
    selection-background-color: rgb(237, 108, 50);
    show-decoration-selected: 1;
    margin-left: -10px;
    padding-left: 15px;
}

QListView::item:hover{
    background-color: rgb(237, 108, 50);
    border: none;
}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbSizeDesc">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>28</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Yu Gothic UI</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QSpinBox {
    border: none;
    border-radius: 5px;
    background-color: rgb(255, 255, 255);
    color: rgb(0, 0, 0);
    padding: 5px; 
}

QSpinBox::up-button {
    background-color: rgb(237, 108, 50);
    border: none;
    width: 25px;
    border-top-right-radius: 5px;
}

QSpinBox::down-button {
    background-color: rgb(237, 108, 50);
    border: none;
    width: 25px;
    border-bottom-right-radius: 5px;
}

QSpinBox::up-arrow {
    image: url(package/icons/chevron-up.png);
    width: 14px;
    height: 14px;
}

QSpinBox::down-arrow {
    image: url(package/icons/chevron.png);
    width: 14px;
    height: 14px;
}</string>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="value">
            <number>16</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lbAttachImg">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Fecha del taller</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tfDate">
         <property name="minimumSize">
          <size>
           <width>660</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;
background-color: white;
border-radius: 5px;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hlDate">
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnColorDate">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>28</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Yu Gothic UI</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(237, 108, 50);
border-radius: 10px</string>
           </property>
           <property name="text">
            <string>Color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbFontDate">
           <property name="font">
            <font>
             <family>Yu Gothic UI</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox{
    border: none;
    border-radius: 5px;
    background-color: rgb(255, 255, 255);
    color: rgb(0, 0, 0);
    padding: 5px; 
}

QComboBox::drop-down{
    background-color: rgb(237, 108, 50);
    border: none;
    width: 25px;
    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
}

QComboBox::down-arrow{
    image: url(package/icons/chevron.png);
    width: 14px;
    height: 14px;
}

QListView{
    border: none;
    color: rgb(132, 131, 130);
    background-color: rgb(255, 255, 255);
    font-weight: bold;
    selection-background-color: rgb(237, 108, 50);
    show-decoration-selected: 1;
    margin-left: -10px;
    padding-left: 15px;
}

QListView::item:hover{
    background-color: rgb(237, 108, 50);
    border: none;
}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbSizeDate">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>28</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Yu Gothic UI</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QSpinBox {
    border: none;
    border-radius: 5px;
    background-color: rgb(255, 255, 255);
    color: rgb(0, 0, 0);
    padding: 5px; 
}

QSpinBox::up-button {
    background-color: rgb(237, 108, 50);
    border: none;
    width: 25px;
    border-top-right-radius: 5px;
}

QSpinBox::down-button {
    background-color: rgb(237, 108, 50);
    border: none;
    width: 25px;
    border-bottom-right-radius: 5px;
}

QSpinBox::up-arrow {
    image: url(package/icons/chevron-up.png);
    width: 14px;
    height: 14px;
}

QSpinBox::down-arrow {
    image: url(package/icons/chevron.png);
    width: 14px;
    height: 14px;
}</string>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="value">
            <number>16</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hlButton">
         <property name="leftMargin">
          <number>600</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnAddFont">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Yu Gothic UI</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(93, 93, 93);
border-radius: 10px</string>
           </property>
           <property name="text">
            <string>Añadir fuente</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="topMargin">
          <number>25</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnBack">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Yu Gothic UI</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #242331;
border-radius: 10px</string>
           </property>
           <property name="text">
            <string>Regresar a pantalla anterior</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnNext">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Yu Gothic UI</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(237, 108, 50);
border-radius: 10px</string>
           </property>
           <property name="text">
            <string>Siguiente</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
